zaas-service/src/main/java/org/zowe/apiml/zaas/security/service/token/ApimlAccessTokenProvider.java

```diff
@@ -195,9 +195,13 @@ private void storeSalt(byte[] salt) throws CachingServiceClientException {
     }
 
     public static byte[] generateSalt() {
-        SecureRandom random = new SecureRandom();
         byte[] salt = new byte[16];
-        random.nextBytes(salt);
+        try {
+            SecureRandom.getInstanceStrong().nextBytes(salt);
+            return salt;
+        } catch (NoSuchAlgorithmException e) {
+            log.error("Could not generate salt", e);
+        }
         return salt;
```

Comment:
is this also part of the coverity finding? that it will hide the exception? I guess before it was throwing it, did it?